src/Copy/BrandCopy.elm

```diff
@@ -606,6 +606,52 @@ brandCopy key =
         StoryThree4Message ->
             CopyText ""
 
+        StoryFourTitle ->
+            CopyText "Lucille"
+
+        StoryFourTeaser ->
+            CopyText "Lucille had never needed to complain about her GP surgery, but when her new medication was messed about she had to speak up."
+
+        StoryFourTeaserImageAlt ->
+            CopyText "✨"
+
+        StoryFour1Quote ->
+            CopyText "My surgery is in a little village, I’ve been with them for 9 years and had never had to make a complaint or get cross with them. But I won’t let the status quo stand when things are unfair. So when I was messed around getting the new medication I needed to go about my life normally I did complain."
+
+        StoryFour1ImageAlt ->
+            CopyText "✨"
+
+        StoryFour1Message ->
+            CopyText ""
+
+        StoryFour2Quote ->
+            CopyText "I wanted to know when my medication would be ready, but they have a policy where you can’t phone them. So I left messages. But they said they were too busy to listen to them - it felt like they were incompetent. I went in and I did get the tablets, but the staff member was quite rude. I told them I wanted an explanation from the practice manager or I would complain."
+
+        StoryFour2ImageAlt ->
+            CopyText "✨"
+
+        StoryFour2Message ->
+            CopyText "I think they were snowed under and the GP’s didn’t know what was going on."
```

Comment:
Should it be GPs ? without the apostrophe?

@katquatermass @roseb04 ?

Reply:
That doesn't sound right.